src/components/Panel/PanelHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC } from 'react';
+
+import { PanelHeaderProps } from './Panel.types';
+
+import { SystemUIButton } from '../Button';
+import { IconName } from '../Icon';
+
+import styles from './panel.module.scss';
+
+export const PanelHeader: FC<PanelHeaderProps> = ({
+    onClose,
+    title,
+    onNoodles,
+    onMaximise,
+}) => {
+    return (
+        <div className={styles.logoGradientHeaderWrapper}>
+            <div className={styles.headerTitle}>{title}</div>
+            <div className={styles.headerActionButtons}>
+                {!!onNoodles && (
+                    <SystemUIButton
+                        iconProps={{
+                            path: IconName.mdiNoodles,
+                            color: 'var(--white-color)',
+                        }}
+                        onClick={onNoodles}
+                        transparent
+                    />
+                )}
+                {!!onMaximise && (
+                    <SystemUIButton
+                        iconProps={{
+                            path: IconName.mdiArrowExpand,
+                            color: 'var(--white-color)',
+                        }}
+                        onClick={onMaximise}
+                        transparent
+                    />
+                )}
+                {!!onClose && (
+                    <SystemUIButton
+                        iconProps={{
+                            path: IconName.mdiClose,
+                            color: 'var(--white-color)',
+                        }}
+                        ariaLabel={'Close'}
+                        onClick={onClose}
+                        transparent
+                    />
+                )}
```

Comment:
{actionButtonThreeProps && (
                        <NeutralButton {...actionButtonThreeProps} />
                    )}
                    {actionButtonTwoProps && (
                        <NeutralButton {...actionButtonTwoProps} />
                    )}
                    {actionButtonOneProps && (
                        <NeutralButton {...actionButtonOneProps} />
                    )}
                    {closable && (
                        <NeutralButton
                            iconProps={{ path: closeIcon }}
                            ariaLabel={'Close'}
                            onClick={onClose}
                            {...closeButtonProps}
                        />
                    )}
                    
 can we use this pattern instead.

Reply:
@ychhabra-eightfold  Are you saying  to make the individual buttons to be controlled outside like, to pass custom icon etc?